README.md

```
@@ -129,28 +168,21 @@ smsgate status zXDYfTmTVf3iMd16zzdBj
-```
-
-If you prefer not to install the CLI tool locally, you can use Docker to run it:
-
-```bash
-docker run -it --rm --env-file .env ghcr.io/android-sms-gateway/cli send --phone '+19162255887' 'Hello, Dr. Turk!'
+smsgate send -u <username> -p <password> --phones '+12025550123' 'Hello, Dr. Turk!'
 ```
 
 #### Output formats
 
 **Text**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)